pkg/plaid_cli/linker.go

```diff
@@ -5,44 +5,75 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"os"
 	"text/template"
 
 	"github.com/plaid/plaid-go/plaid"
 	"github.com/skratchdot/open-golang/open"
 )
 
 type Linker struct {
-	Results    chan string
-	Errors     chan error
-	Client     *plaid.Client
-	ClientOpts plaid.ClientOptions
-	Data       *Data
+	Results       chan string
+	RelinkResults chan bool
+	Errors        chan error
+	Client        *plaid.Client
+	ClientOpts    plaid.ClientOptions
+	Data          *Data
 }
 
 type TokenPair struct {
 	ItemID      string
 	AccessToken string
 }
 
-func (l *Linker) Relink(itemID string, port string) (*TokenPair, error) {
+func (l *Linker) Relink(itemID string, port string) error {
 	token := l.Data.Tokens[itemID]
-	res, err := l.Client.CreatePublicToken(token)
+	hostname, err := os.Hostname()
```

Comment:
The LinkToken API requires a unique user id. I figured we can just use the hostname here, but maybe we want this to be something the user can configure? What do you think?